introduction_to_amazon_algorithms/jumpstart-foundation-models/llama3.1-fine-tuning.ipynb

```diff
@@ -0,0 +1,707 @@
+{
```

Comment:
<p>Remove "please".</p><p> </p><p>Call out this is gated model since it requires HF token to access model's ....</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/4729/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>